test/JDBC/expected/BABEL-4926.out

```diff
@@ -0,0 +1,187 @@
+-- create a new table BABEL_4926 1
+create table BABEL_4926(ID int);
+go
+select ID from BABEL_4926;
+go
+~~START~~
+int
+~~END~~
+
+-- Create a new table BABEL_4926_REPO with column ID 
+select ID into BABEL_4926_REPO from BABEL_4926;
+go
+-- Indexing over column ID
+create index IDX_REPRODUCTION on BABEL_4926_REPO(ID);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions as ID from pg_attribute where attrelid = (select oid from pg_class where relname = 'babel_4926_repo');
+go
+~~START~~
+varchar#!#varchar
+tableoid#!#<NULL>
+cmax#!#<NULL>
+xmax#!#<NULL>
+cmin#!#<NULL>
+xmin#!#<NULL>
+ctid#!#<NULL>
+id#!#{bbf_original_name=ID}
+~~END~~
+
+-- dropping the table BABEL_4926
+drop table BABEL_4926;
+go
+-- dropping the table BABEL_4926_REPO
+drop table BABEL_4926_REPO;
+go
+
+
+
+-- create table with multiple columns with both mixed case names 2
+create table BABEL_4926(numb int,Numb1 int,ColuM int);
+go
+-- select the columns from the table
+select numb,Numb1,ColuM from BABEL_4926;
+go
+~~START~~
+int#!#int#!#int
+~~END~~
+
+-- create the BABEL_4926_REPO and select the column from BABEL_4926 
+select numb,Numb1,ColuM into BABEL_4926_REPO from BABEL_4926;
+go
+-- indexing over the columns of BABEL_4926_REPO
+create index IDX_REPRODUCTION on BABEL_4926_REPO(numb,Numb1,ColuM);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions from pg_attribute where attrelid = (select oid from pg_class where relname = 'babel_4926_repo');
```

Comment:
also prefix column name with jira name. So that we can only select only those column here.